COPYRIGHT-NOTICES.md

```diff
@@ -0,0 +1,138 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+# Copyrights and OpenJS Foundation projects
+
+## Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the DCO or a CLA,
+the original copyright holders retain their copyrights.
+
+## Copyright Notices
+
+OpenJS Foundation does not require or recommend that every contributor include their
+copyright notice in contributed files. [See below for more details on why
+not.](#why-not-list-every-copyright-holder)
+
+Instead, OpenJS Foundation recommends using a more general statement in a form similar to the
+following (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
+
+These statements are intended to communicate the following:
+- the work is copyrighted;
+- the contributors of the code licensed it, but retain ownership of their copyrights; and
+- it was licensed for distribution as part of the named project.
+
+By using a common format, the project avoids having to deal with names of
+copyright holders, years or ranges of years, and variations on the (c) symbol.
+This aims to minimize the burden on developers and maintainers as well as
+redistributors of the code.
```

Comment:
Should projects encourage .git/hooks/pre-commit and/or use CI/CD checks to flag content that doesn't comply.  Is this something that could be templatized?

Reply:
I would be inclined to agree with @boneskull here, changes to these lines should be a rarity.

Reply:
Are these required on all source files?

I agree that, once present, they're unlikely to change, but if they are per-file boilerplate, then a linter might be useful for new source files.

Reply:
Please let's not require the ancient and obsolete-in-the-wider-ecosystem practice of per-file boilerplate.

Reply:
Yep, correct, per the guidance, new lines aren't needed per-file (and in fact, aren't needed at all unless you choose to add one).  But once someone else's line is there, do not change or remove it.

Reply:
What about copyright mentioning the JS Foundation?

In webhint's case we [have this](https://github.com/webhintio/hint/blob/2473518d01a22f021fb0db16692f9f67c8941eb5/LICENSE.txt#L189) in the License:

> Copyright JS Foundation and other contributors, https://js.foundation

Should it be updated to the OpenJS Foundation?

Reply:
@molant I'm checking on this specific situation, and will get back to you when I get an answer.  Thanks for raising it.

Reply:
@molant Ok, got an answer for you.

Because JS Foundation merged into the OpenJS Foundation, it would be factually correct to update this to OpenJS Foundation.  However, since these notices are informational in nature and optional, it is up to the maintainer of the file whether to update this specific line.

In other words, it can't hurt to update it, but it is not required either.

Does this help?

Reply:
@brianwarner @ljharb If the lawyercats agree that per-file (C) boilerplate is unnecessary and that having it on old files and not putting it on new doesn't put the new files in a different legal category than the old, that addresses my concerns.

Reply:
@mikesamuel Yep, that's correct.

Reply:
> Does this help?

Yes, thanks a lot!